pages/application.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+
+import { handleLoginRedirect, getProfile } from "../lib/authenticate";
+
+import Head from "../components/Head";
+import Navbar from "../components/Navbar";
+import Footer from "../components/Footer";
+
+import { Background, Container } from "../styles";
+
+import Step from "../components/steps/Profile";
+
+const Application = ({ profile }) => {
+  return (
+    <>
+      <Head title="HackSC Odyssey - Application" />
+      <Navbar loggedIn />
+      <Background>
+        <Container>{profile && <Step profile={profile} />}</Container>
+      </Background>
+      <Footer />
+    </>
+  );
+};
+
+Application.getInitialProps = async ({ req }) => {
+  const profile = await getProfile(req);
+
+  // Null profile means user is not logged in
```

Comment:
Just want to double check this is true, since it seems like you could be logged in and not have started a profile

Reply:
Yeah, the first ever call to `GET /api/profile` should create a blank one so that should be handled by 27 -- I'm prob going to spend some time today writing some automated tests to verify flows and stuff